Database/schema.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE games(
+    id serial PRIMARY KEY,
+    multiplayer varchar(255),
+    last_palayed_at DATE,
+    publish_date Date,
+    archived Boolean,
+)
+
+CREATE TABLE authors(
+    id serial PRIMARY KEY,
+    first_name varchar(255),
+    last_name varchar(255),
+)
+
+CREATE TABLE "genre"(
+    "id" INT PRIMARY KEY,
+    "name" VARCHAR(255)
+);
+
+CREATE TABLE "music_albums"(
+    "id" INT PRIMARY KEY,
+    "on_spotify" BOOLEAN,
+    "genre" INT,
+    "author" INT,
+    "source" INT,
+    "label" INT,
+    "publish_date" DATE,
+    "archived" BOOLEAN
+);
+
+CREATE TABLE labels (
+  id serial PRIMARY KEY,
+  title VARCHAR(200) NOT NULL,
+  color VARCHAR(200) NOT NULL,
+  PRIMARY KEY (id)
+)
+
+CREATE TABLE books (
+  id serial PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  genre_id INT,
+  author_id INT,
+  label_id INT,
+  PRIMARY KEY (id)
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id),  
```

Comment:
- [ ] Good job working on the database but please make sure all foreign keys columns have corresponding indexes.